packages/plugins/docs-ui/src/lib/components/links/document-links-panel.component.html

```
@@ -0,0 +1,83 @@
+	The action gates are `*ngIf` over Store checks rather than `*ngxPermissionsOnly`:
+	uploading needs DOCS_CREATE **and** DOCS_UPDATE (a document write followed by a
+	link write), and `ngxPermissionsOnly` with a list is an OR — it would offer the
+	button to someone who can only do half of it and leave the file unattached.
+-->
+<nb-card *ngIf="visible" class="docs-links-panel">
+	<nb-card-header class="docs-links-panel-header">
+		<span>{{ 'DOCS.LINKS.PANEL_TITLE' | translate }}</span>
+		<span class="docs-links-panel-count" *ngIf="links.length">{{ links.length }}</span>
+
+		<span class="docs-links-panel-actions">
+			<button *ngIf="canLink" nbButton ghost size="tiny" (click)="attachExisting()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)